include/gigamonkey/p2p/AbstractAddressManager.hpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2022 Katrina Knight
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef GIGAMONKEY_P2P_ABSTRACTADDRESSMANAGER_HPP_
+#define GIGAMONKEY_P2P_ABSTRACTADDRESSMANAGER_HPP_
+
+#include <string>
+#include <list>
+#include <boost/heap/priority_queue.hpp>
+#include "data/encoding/endian/arithmetic.hpp"
+#include "address.hpp"
+namespace Gigamonkey::Bitcoin::P2P {
+	struct NodeAddress {
```

Comment:
types have lower cases names like "node_address". Does it need node? Could it simply be called "address" ?